OliveVSIX/NugetPacker/NugetPackerLogic.cs

```diff
@@ -95,13 +95,24 @@ static void GenerateNugetFromVSProject(string projectAddress)
 
             if (TryPackDotnet(projectAddress, out string packingMessageDotnet))
             {
-                if (!TryPush(packageFilename, out string pushingMessage))
```

Comment:
@ashkan-saeedi-mazdeh but I think there shouldn't be any difference between `dotnet` or `nuget` package as they both generate the same nuget package file!

so I believe if we have an issue for sending nuget package there should be something else!

Reply:
@iPazooki  What I've done is, I checked what Microsoft docs say about nuget packages for .NET core and .NET Standard
Checked the code in the VSIX and changed what was different.

I don't know what the issue has been and could not test it due to not having nuget API keys

I did not create the `dotnet pack` command and it was there and should be used for building nuget packages as Microsoft docs say, If nuget.exe can be used for it regardless of what the docs said, that's what I'm not aware of.

About the push command , probably both should be fine unless they do different types of analysis on the packages which should be different.


If you have any info/documentation links which I might have missed or if you know anything more about the issue we have with packing and pushing .csproj based projects , I would be more than happy to know more